src/Person/Person.jsx

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
You didn't import react, Person.scss file and proptypes

Reply:
true story...